fairseq/models/speech_to_text/convtransformer.py

```diff
@@ -171,7 +171,7 @@ def build_encoder(cls, args):
     @classmethod
     def build_decoder(cls, args, task, embed_tokens):
         decoder = TransformerDecoderNoExtra(args, task.target_dictionary, embed_tokens)
-        if getattr(args, "load_pretrained_decoder_from", None):
+        if getattr(args, "load_pretrained_decoder_from", None) is not None:
```

Comment:
nit: is not None condition check is not needed I guess?
```suggestion
        if getattr(args, "load_pretrained_encoder_from", None):
```

Reply:
@sravyapopuri388 I just followed Google Python Style Guide.

Reply:
@sravyapopuri388 wdyt?